docs/how_to/install_puppet_enterprise.md

```diff
@@ -0,0 +1,107 @@
+# How-To Install Puppet Enterprise
```

Comment:
So, I don't think something that is called a "reference" doesn't mean it's "complete", but ultimately I don't care if you use the term reference or not.  I'm just not a huge fan of the `how-to` redundancy. Removing the hyphen seems better to me.